src/scripts/marcRecordUi.js

```diff
@@ -336,55 +337,265 @@ export function filterField(field) {
   }
 }
 
+export function getPreviousEditableSibling(elem) { // melinda-ui-commons?
+  const prevElem = elem.previousSibling;
+  if (!prevElem) {
+    //console.log("  NO PREV");
+    return prevElem;
+  }
+  if (isEditableDiv(prevElem)) {
+    //console.log("  IT'S ME");
```

Comment:
Some console logs could be removed

Reply:
OR change to more permanent debug style

Reply:
I thought that these would show up in the browser which is where I want them... (Elsewhere I've used console.log() in places where I shouldn't.)